Include/cpython/pystate.h

```diff
@@ -6,7 +6,7 @@
 extern "C" {
 #endif
 
-#include "cpython/initconfig.h"
```

Comment:
cpython/ subdirectory should not be in the list of include directory.

Reply:
I agree, but there's no need to put cpython in the list of include directories. pystate.h and inticonfig.h are in the same directory which is the first path searched by the preprocessor. At least this is true according to Wikipedia, GNU docs and visual studio docs. Prior to this change, you needed to update the default include path to compile the following app. 

    #include "include/python3.9/Python.h"
    int main() {}

This now compiles without having to update the default include paths.

    gcc main.c

Reply:
This was especially useful prior to python 3.8, you could have multiple files that look like

    #include "deps/python3.7/Python.h"
    get_python_37_data();

and

    #include "deps/python3.6/Python.h"
    get_python_36_data();

and at runtime determine which version of get_python_xx_data() to use. These would both compile without specialising your include path for each source file.

Reply:
IMO it's not the right approach: https://bugs.python.org/issue39026#msg369242